src/stores/skjema-tilgang-store.ts

```diff
@@ -0,0 +1,14 @@
+import createUseContext from 'constate';
```

Comment:
Bra med egen tilgang store for isReadOnly, og setIsReadOnly, men tenker at ting som har å gjøre med innlogget veileder kan trekkes ut i egen store.

Reply:
Tenker om å bytte isReadyOnly med erInnloggetAnsvarligVeileder og fjerne isReadyOnly fra alle steder i filer.

Reply:
Byttet  !isReadOnly med erInnloggetAnsvarligVeileder og isReadyOnly med    !erInnloggetAnsvarligVeileder